src/components/Logs/Table/logs-table.js

```diff
@@ -15,111 +22,164 @@ export function LogsTable({ logs, actions = [] }) {
-            ))}
+            </div>
+          );
+        },
+        accessor: "message",
+        id: "message",
+        Cell: function messageCell({ cell: { value } }) {
+          return (
+            <div
+              className="break-all"
+              // eslint-disable-next-line react/no-danger
+              dangerouslySetInnerHTML={{
+                __html: DOMPurify.sanitize(convert.toHtml(value || ""))
+              }}
+            />
+          );
+        }
+      }
+    ],
+    []
+  );
+  const data = useMemo(() => logs, [logs]);
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    rows,
+    prepareRow,
+    selectedFlatRows,
+    state: { selectedRowIds }
+  } = useTable(
+    {
+      columns,
+      data
+    },
+    useRowSelect,
+    (hooks) => {
+      hooks.visibleColumns.push((columns) => [...columns]);
+    }
+  );
+  return (
+    <div className="mx-auto flex flex-col justify-center relative">
+      {actions.length >= 0 && (
+        <div
+          className="flex justify-between items-center overflow-y-hidden mb-4"
+          style={{
+            maxHeight: Object.keys(selectedRowIds).length ? "100px" : "0px",
+            transition: "max-height 0.25s  ease-in-out"
+          }}
+        >
+          <div>
+            {Object.keys(selectedRowIds).length > 0 && (
+              <span className="text-sm text-gray-400">
+                {Object.keys(selectedRowIds).length} selected
+              </span>
+            )}
           </div>
-        )}
 
-        <div className="pb-12">
-          <div className="flex flex-col">
-            <div className="align-middle inline-block min-w-full">
-              <table
-                className="min-w-full divide-y divide-gray-300 border-separate"
-                style={{ borderSpacing: "0px" }}
-              >
-                <thead className="sticky top-0 bg-white">
-                  <tr>
-                    <th className="border-b border-gray-300" />
-                    <th className="text-left text-xs font-medium text-gray-500 uppercase tracking-wider border-b border-gray-300 pr-5">
-                      Timestamp
-                    </th>
-                    <th className="text-left text-xs font-medium text-gray-500 uppercase tracking-wider border-b border-gray-300">
-                      Message
+          {actions.map((action) => (
+            <button
+              key={action.label}
+              type="button"
+              disabled={!Object.keys(selectedRowIds).length}
+              onClick={() => {
+                action.handler(selectedFlatRows.map((d) => d.original));
+              }}
+              className={clsx(
+                Object.keys(selectedRowIds).length > 0
+                  ? "text-indigo-700 bg-indigo-100 hover:bg-indigo-200"
+                  : "text-gray-400 bg-gray-200",
+                "inline-flex items-center px-2.5 py-1.5 border border-transparent text-xs font-medium rounded"
```

Comment:
please use btn-primary and btn-disabled classes

Reply:
removed this block of code, but used these classes in another place